.github/workflows/ci-python.yml

```diff
@@ -1,14 +1,34 @@
-name: Continuous Integration - Python
+name: CI - Python
 
 on:
-  push:
-    branches:
-      - main
   pull_request:
+  push:
+    branches: [main]
 
 jobs:
-  build-and-test-python:
-    name: Build + unit tests
+  python-runtime-go-tests:
+    name: Python runtime Go tests
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v4
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version-file: go.mod
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.12"
```

Comment:
try to use some ref here

Reply:
No option for that in GH workflow files, but we'll migrate to 3.12 throughout the repo, and add a reference to the "source of truth"